## tutorial --errors and exception raising

def main():

	try:
		a = 1
		b = 0
		c = a/b
		print("ayaaaa")
	except Exception:
		print("Exception encountered")

	print("Debug: yha to aaya h bhai")

if __name__ == '__main__':
	main()


############## 1 ##########

# def main():
	
# 	try:
# 		a = 1
# 		b = 0
# 		c = a/b
# 		print("ayaaaa")
# 	except Exception:
# 		print("Exception encountered")

# if __name__ == '__main__':
# 	main()